### DISCUSSION TITLE
File with content shown as an empty file

### DISCUSSION THREAD

**Discussion by wuyscc** [nvim-tree/nvim-tree.lua#2207] [Q&A] [answered]
### Description

When I using nvim-tree to open my SvelteKit project, it unable to open files in folder such as `(app)` or `(blog)`

### Neovim version

```
NVIM v0.9.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

### Operating system and version

Windows 10 22H2

### nvim-tree version

lastest

### Minimal config

```
require("nvim-tree").setup {
  sort_by = "case_sensitive",
  view = {
    width = 25
  },
  view = {
    side = "right"
  },
  renderer = {
    group_empty = true
  },
  filters = {
    dotfiles = true
  }
}
```

### Step to reproduce

1. Create a folder named `(blog)`
2. Create a file with content in `(blog)` folder
3. Open file in `(blog)` folder

### Expected behavior

File opened with content shown

### Actual behavior

File can't be opened or opened with empty content

### Screenshots about error

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/42114171/9cb37bc2-1acf-4e4d-b0b7-7ae9bc21c770)

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/42114171/ae6266aa-aaab-4380-a613-c8aa202ddc51)

**Comment by alex-courtis** [comment, marked as answer]
Windows is always a problem; this cannot be reproduced under linux.

It looks like you're using powershell. Please attempt under WSL and raise a new [bug report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with your findings.

Please note that nvim-tree developers do not have access to or expertise with windows. A pull request is the pragmatic means of resolving this, see [Development](https://github.com/nvim-tree/nvim-tree.lua/wiki/Development)
